libs/@guardian/source-react-components/scripts/create-icons/create-icon-component.ts

```diff
@@ -117,10 +117,12 @@ export const createIconComponent = async ({
 		},
 	);
 
-	const label = labels[icon.name];
+	const label = labels[icon.name] ?? icon.name;
 
 	if (!label) {
-		throw new Error(`No label found for ${icon.name}`);
+		console.warn(
```

Comment:
Likewise here - do we need to change this?

Reply:
The icons in Figma don’t have labels so by throwing here the whole script stopped and I couldn’t complete the update.

Reasoning was that we could warn about the missing label and fall back to the icon id, so that the update could still go ahead

Reply:
(Manually added human readable labels after to the three new icons as I don’t have Figma edit access to add them at source)

Reply:
Thanks for the explanation - is there a way that we can get someone with Figma edit access to add the labels at source?

Reply:
Feels a bit like we might be changing the system here when the issue might be more about access to Figma

Reply:
Looking into why the label setup didn't work as-is in Figma. Agree that changing the system could be wrong as the enforcement here ensures that the right label is used.. In terms of this specific case, it does feel like the system is being quite inflexible by throwing. Debugging this and writing a fix yday to get the script to run cost me about 15-20 mins total, so I feel like capturing the change in some form might be of value to people who come to update icons in the future. How to capture it is an interesting question though :)

Reply:
Yeah its an interesting point - is a comment to say "if this is throwing, check you have the correct label in Figma" too simple?

Reply:
yeah that's a good shout in this case I think - just doing some digging because it seems as though the label is there, but wasn't being picked up. Either way as this is seems to be an easy point of failure, the comment seems valuable. 

I'll back out the code change, leave the comment, and look into what's going wrong with the Figma integration.

This kind of integration seems really prone to this kind of breakage because of how flexible Figma is as a platform; so easy for someone to accidentally break the contract by moving a label outside of a frame or something.

Reply:
Nice one, cheers.